AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiugne stile e html per adattare a mockup.

ninomarrazzo opened the pull request.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup. (hqCfvHZy) - Aggiugne stile e html per adattare a mockup; in corso.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup. (hqCfvHZy) - Aggiugne stile e html per adattare a mockup; in corso.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup. (hqCfvHZy) - Aggiunge stile link accesso al forum; in corso.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup. (hqCfvHZy) - Aggiunge stile link accesso al forum; in corso.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup. (hqCfvHZy) - Aggiunge stile link accesso al forum.

pdavide reviewed (changes requested):
<img width="676" alt="schermata 2018-10-11 alle 11 49 34" src="https://user-images.githubusercontent.com/9119858/46797151-aa1d7300-cd4e-11e8-8982-de53ad0c6ab0.png">

- Manca il segno +/- per espandere/collassare i commenti
- Manca la freccia per espandere/collassare i singoli commenti
- Il testo dovrebbe essere sans-serif

pdavide reviewed (changes requested):
<img width="642" alt="schermata 2018-10-11 alle 12 20 25" src="https://user-images.githubusercontent.com/9119858/46797723-3aa88300-cd50-11e8-8ccf-46bc03edce07.png">

- Il testo interno alla textarea non dovrebbe essere bold.

<img width="773" alt="schermata 2018-10-11 alle 12 34 05" src="https://user-images.githubusercontent.com/9119858/46798349-fd44f500-cd51-11e8-8192-5e7290bde586.png">

- La textarea dovrebbe avere una `min-height: 5em;`
- I bottone per inviare il commento dovrebbe avere lo [stile "primary" di bootstrap italia](https://italia.github.io/bootstrap-italia/docs/componenti/bottoni/#small).
- Il bottone "annulla" può essere eliminato
- Il testo di aiuto per la sintassi markdown dovrebbe essere contenuto in un [popover che si apre verso il basso](https://italia.github.io/bootstrap-italia/docs/componenti/popover/#le-quattro-direzioni) cliccando su un bottone "suggerimenti".
- Il bottone "suggerimenti" dovrebbe avere lo [stile "secondary outline" di bootstrap italia](https://italia.github.io/bootstrap-italia/docs/componenti/bottoni/).
- Il testo placeholder dentro la textarea dovrebbe essere: "Scrivi qui un commento. Per formattare il testo puoi usare Markdown, clicca sul bottone "suggerimenti" per vedere degli esempi."
- Il testo "Caratteri richiesti: -20" dovrebbe avere una `font-size: 0.8em` ed essere visualizzato sulla stessa riga dei bottoni allineato a destra. Il testo dovrebbe essere "Scrivi almeno altri <num> caratteri" e poi non dovrebbe essere visibile superata la soglia richiesta.
- Al click sul bottone invia, il testo non dovrebbe essere coperto e non si dovrebbe visualizzare "Invio del commento", invece il bottone e la textarea dovrebbero passare in modalità "disabled" e dovrebbe apparire uno spin loader a destra del bottone "suggerimenti" (va bene lo stesso usato per il popup).

pdavide reviewed (changes requested):
<img width="761" alt="schermata 2018-10-11 alle 12 50 30" src="https://user-images.githubusercontent.com/9119858/46799434-0daa9f00-cd55-11e8-8498-3113cd67434a.png">

Il testo "Contenuto sembra poco chiaro, è una frase completa?" non dovrebbe apparire più di una volta in caso di più invii e dovrebbe essere: "Il tuo commento non sembra abbastanza comprensibile. Prova con qualcosa di semplice e lineare."

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiugne stile e html per icone docs-icon.

ninomarrazzo reviewed (commented):
Ho apportato la modifica anche alle icone. Ora sono correttamente visualizzate. Verifico la questione della frase di errore "contenuto poco chiaro" e rieffettuo push.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Fix errore post ripetuto.

ninomarrazzo reviewed (commented):
Fixato anche il problema della frase di errore che si ripeteva ad ogni submit. Implemento il resto delle richieste.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiunge pulsante suggerimenti; Migliora stile commenti - textarea.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiunge nuovo placeholder in textarea.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiunge nuovo stile per i caratteri richiesti.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiunge nuovo stile per i caratteri richiesti.

ninomarrazzo committed:
AGID-475 Merge bootstrap-italia-forum

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiunge nuovo stile per gestire loader all'invio di un nuovo commento.

ninomarrazzo commented:
@pdavide Ho ultimato tutte le modifiche richieste. Il codice di questa PR dovrà essere ritoccato a seguito del merge di https://github.com/italia/docs-italia-theme/pull/140.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Aggiunge nuovo stile per gestire loader all'invio di un nuovo commento.

pdavide reviewed (changes requested):
Ti segnalo qualche ulteriore piccola rifinitura (per lo più dovute a mie imprecisioni):
- il testo dei commenti visualizzati, della data e del nome dell'organizzazione dovrebbero essere sans-serif
- Il bottone "invia" dovrebbe essere a sx e "suggerimenti" a dx
- La textarea dovrebbe avere la class `p-2` al posto di `pl-3` e `pr-3`
- Lo spin loader non dovrebbe essere interno al bottone "invia" ma esterno a destra del bottone "suggerimenti"

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup. (hqCfvHZy) - Fix vari

ninomarrazzo committed:
AGID-475 Merge bootstrap-italia-forum

ninomarrazzo committed:
AGID-520 Verificare il funzionamento dell'integrazione con il forum anche su dispositivi mobile. (rYD7Wqsr) - Fix post-merge.

ninomarrazzo committed:
AGID-520 Verificare il funzionamento dell'integrazione con il forum anche su dispositivi mobile. (rYD7Wqsr) - Fix post-merge.

ninomarrazzo committed:
AGID-520 Verificare il funzionamento dell'integrazione con il forum anche su dispositivi mobile. (rYD7Wqsr) - Fix post-merge.

ninomarrazzo reviewed (commented):
Dovremmo esserci!

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Stili dei commenti.

ninomarrazzo committed:
AGID-475 Allineare lo stile dei commenti a quanto presente nel mockup (hqCfvHZy) - Stili dei commenti; font sans-serif.

ninomarrazzo reviewed (commented):
Ho apportato le modifiche richieste.

pdavide committed:
Add user icon next to login link

pdavide committed:
Remove not needed post parameter

pdavide committed:
Reverse post order (newer first)

pdavide committed:
Remove unused function + remove timeout + close popover on blur

pdavide committed:
Update style

- remove unused css rules
- use sans font in the whole comments block
- add some margin to the submit button

pdavide committed:
Merge branch 'bootstrap-italia-forum' into FEATURE/AGID-475
